_global/js/board_front.js

```diff
@@ -39,13 +42,24 @@ function addImages(board) {
     const spiritImage = board.getAttribute('spirit-image');
 
     const spiritBorder = board.getAttribute('spirit-border');
+	
+    const imageSize = board.getAttribute('spirit-image-scale');
 
+    
+	const specialRules = board.querySelectorAll('special-rules-container')[0]
+    let height = specialRules.getAttribute('height')
+	    if(!height){
+        const computedStyle = window.getComputedStyle(specialRules)
+        height = computedStyle.getPropertyValue('height')
+    }
+	
     if(spiritBorder){
         const specialRules = board.querySelectorAll('special-rules-container')[0]
         specialRules.innerHTML = `<div class="spirit-border" style="background-image: url(${spiritBorder});" ></div>` + specialRules.innerHTML
     }
     if(spiritImage){
-        board.innerHTML = `<div class="spirit-image" style="background-image: url(${spiritImage});" ></div>` + board.innerHTML
+		//Image now scales to fill gap. 'imageSize' allows the user to specify what % of the gap to cover
+        board.innerHTML = `<div class="spirit-image" style="background-image: url(${spiritImage}); background-size: auto ${imageSize}; height:calc(100% - ${height}); width:1700px;" ></div>` + board.innerHTML
```

Comment:
The 'width:1700px' here is arbitrary and I'm open to other fixes. I'd like the width to be unbounded but I wasn't able to find code that would do so and still work with the image scaling.